debian/xpu-smi.install

```diff
@@ -1 +1,2 @@
 usr/bin
```

Comment:
/usr/bin/xpu-smi

Reply:
The assumption being that any new binaries we add would be installed via a different package? That's currently the only file in usr/bin, so I just want to make sure before every new binary requires a new change to the deb install process